litgpt/model.py

```diff
@@ -375,7 +446,7 @@ def forward(
         y = self.scaled_dot_product_attention(q, k, v, mask)
 
         # Re-assemble all head outputs side by side.
-        y = y.reshape(B, T, self.config.head_size * self.config.n_head)
+        y = y.flatten(start_dim=-2, end_dim=-1)
```

Comment:
Here it's a bit over-engineered 🙂.

Overall, I think that the existing one is easier to read and gives you info of the final shape.

Reply:
OK